lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/TsNode.cpp

```diff
@@ -0,0 +1,24 @@
+#include "lazy_tensor_core/csrc/ts_backend/TsNode.h"
+
+namespace torch_lazy_tensors {
+namespace ir {
+
+TsNode::TsNode(OpKind op, OpList operands, lazy_tensors::Shape shape,
+               size_t num_outputs, lazy_tensors::hash_t hash_seed)
```

Comment:
it's so weird an IR node needs to worry about `hash_seed` @wconstab  @alanwaketan  do you guys know if we ever pass anything different from the default value in there?

Reply:
oh, timely comment! I happen to have just renamed all of this on my local copy.  I agree, it was a bit of a misnomer.  In this case, the 'seed' is really a partial hash passed in by the leaf class, which hashes over any scalars/constants provided to the leaf ctor.